### PR TITLE
WF-63246-Add dependent NuGet packages details for PPTX to PDF conversion in Linux OS

### PR DESCRIPTION
**Pull request by AntonihilSahayaraj**
<p>Kindly review the below merge request and let me know your feedback.</p>
<table>
<tbody>
<tr>
<td>
<p>Task Link</p>
</td>
<td>
<p><a href="https://syncfusion.atlassian.net/browse/WF-63246">https://syncfusion.atlassian.net/browse/WF-63246</a></p>
</td>
</tr>
<tr>
<td>
<p>Description</p>
</td>
<td>
<p>Add dependent NuGet packages details for PPTX to PDF conversion in Linux OS</p>
</td>
</tr>
<tr>
<td>
<p>Solution</p>
</td>
<td>
<p>Added content to what are the dependent NuGet packages details for PPTX to PDF conversion in Linux OS</p>
</td>
</tr>
</tbody>
</table>
<p><br /><br /></p>

### CODE REVIEW

File: File-Formats/Presentation/FAQ.md [hunk 1]
```diff
@@ -46,3 +46,56 @@ documentation: UG
...
+{{'[SkiaSharp.NativeAssets.Linux v2.80.2 NuGet](https://www.nuget.org/packages/SkiaSharp.NativeAssets.Linux/2.80.2)'| markdownify }}<br/>
+</td>
+</tr>
+<tr>
+<td>
+Before v18.4.0.x
+</td>
+<td>
+Install SkiaSharp.Linux NuGet package for .Net Core application in Linux OS. you can find the SkiaSharp.Linux NuGet package created by us from {{'[here](https://www.syncfusion.com/downloads/support/directtrac/general/ze/SkiaSharp.Linux.1.59.3-2103435070)'| markdownify }}.<br/>
+For more information, Please refer [here](https://help.syncfusion.com/file-formats/docio/faq#how-to-perform-word-to-pdf-in-linux-prior-to-v184-release).
```
Comment (MohanaselvamJothi) [inline File-Formats/Presentation/FAQ.md:98]: Can you please remove this link or add equivalent link for presentation

File: File-Formats/Presentation/FAQ.md [hunk 2]
```diff
@@ -46,3 +46,56 @@ documentation: UG
    
    You can improve the quality of converted images by specifying the image resolution. Refer – [Converting PowerPoint presentation to Images](/file-formats/presentation/getting-started#converting-powerpoint-presentation-to-images)
 
+10. ## What are the NuGet packages to be installed to perform PowerPoint Presentation to PDF conversion in Linux OS?
```
Comment (MohanaselvamJothi) [inline File-Formats/Presentation/FAQ.md:49]: Please add this FAQ link in PPT to PDF conversion page

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/Presentation/FAQ.md [hunk 3]
```diff
@@ -46,3 +46,56 @@ documentation: UG
    
    You can improve the quality of converted images by specifying the image resolution. Refer – [Converting PowerPoint presentation to Images](/file-formats/presentation/getting-started#converting-powerpoint-presentation-to-images)
```
Comment (Chrispineagunja) [inline File-Formats/Presentation/FAQ.md:47]: Improve the quality of converted images by specifying the image resolution. Refer to [Converting PowerPoint presentation to Images](/file-formats/presentation/getting-started#converting-powerpoint-presentation-to-images).

File: File-Formats/Presentation/FAQ.md [hunk 4]
```diff
@@ -46,3 +46,56 @@ documentation: UG
...
 
+10. ## What are the NuGet packages to be installed to perform PowerPoint Presentation to PDF conversion in Linux OS?
+
+In Linux OS, you can perform PowerPoint Presentation to PDF conversion using .NET Core (Targeting .netcoreapp) application. You can refer [PowerPoint Presentation to PDF conversion NuGet packages](https://help.syncfusion.com/file-formats/presentation/nuget-packages-required#converting-powerpoint-presentation-into-pdf) to know about the packages required to deploy .NET Core (Targeting .netcoreapp) applications with PowerPoint Presentation to PDF conversion capabilities.
```
Comment (Chrispineagunja) [inline File-Formats/Presentation/FAQ.md:51]: In Linux OS, perform PowerPoint Presentation to PDF conversion using the .NET Core (Targeting .netcoreapp) application. Refer to [PowerPoint Presentation to PDF conversion NuGet packages](https://help.syncfusion.com/file-formats/presentation/nuget-packages-required#converting-powerpoint-presentation-into-pdf) to know about the packages required to deploy the .NET Core (Targeting .netcoreapp) applications with PowerPoint Presentation to PDF conversion capabilities.

File: File-Formats/Presentation/FAQ.md [hunk 5]
```diff
@@ -46,3 +46,56 @@ documentation: UG
...
+
+In addition to the previous NuGet packages, the following NuGet packages need to be installed in your application.
```
Comment (Chrispineagunja) [inline File-Formats/Presentation/FAQ.md:53]: In addition to the previous NuGet packages, your application needs to install the following NuGet packages.

File: File-Formats/Presentation/FAQ.md [hunk 6]
```diff
@@ -46,3 +46,56 @@ documentation: UG
...
+<td>
+{{'[SkiaSharp.NativeAssets.Linux v2.80.2 NuGet](https://www.nuget.org/packages/SkiaSharp.NativeAssets.Linux/2.80.2)'| markdownify }}<br/>
+</td>
+</tr>
+<tr>
+<td>
+Before v18.4.0.x
+</td>
+<td>
+Install SkiaSharp.Linux NuGet package for .Net Core application in Linux OS. you can find the SkiaSharp.Linux NuGet package created by us from {{'[here](https://www.syncfusion.com/downloads/support/directtrac/general/ze/SkiaSharp.Linux.1.59.3-2103435070)'| markdownify }}.<br/>
```
Comment (Chrispineagunja) [inline File-Formats/Presentation/FAQ.md:97]: Install the SkiaSharp.Linux NuGet package for the .NET Core application in Linux OS. Find the SkiaSharp.Linux NuGet package created by us from {{'[here](https://www.syncfusion.com/downloads/support/directtrac/general/ze/SkiaSharp.Linux.1.59.3-2103435070)'| markdownify }}.<br/>

File: File-Formats/Presentation/FAQ.md (hunk 1 above)
Comment (Chrispineagunja) [inline File-Formats/Presentation/FAQ.md:98]: For more information, Please refer [here](https://help.syncfusion.com/file-formats/docio/faq#how-to-perform-word-to-pdf-in-linux-prior-to-v184-release).

**Review by AntonihilSahayaraj — COMMENTED** [review]
Feedbacks are addressed

File: File-Formats/Presentation/FAQ.md (hunk 3 above)
Comment (AntonihilSahayaraj) [reply File-Formats/Presentation/FAQ.md:47]: It's already available in UG

File: File-Formats/Presentation/FAQ.md (hunk 2 above)
Comment (AntonihilSahayaraj) [reply File-Formats/Presentation/FAQ.md:49]: Once FAQ MR merged. I will create separate MR for that

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**